templates/docs/examples/layouts/empty-state/user-triggered.html

```diff
@@ -0,0 +1,47 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Empty State / User action triggered  {% endblock %}
+
+{% block standalone_css %}layouts{% endblock %}
+
+{% block content %}
+<div class="p-strip is-shallow u-fixed-width">
+    <h2>
+        Search results for "flibberdygibbery"
+    </h2>
+</div>
+<div class="u-fixed-width">
+    <form class="p-search-box">
+        <input type="search" class="p-search-box__input" name="search" required="" value="flibberdygibbery">
+        <button type="submit" class="p-search-box__button"><i class="p-icon--search"></i></button>
+    </form>
+</div>
+<div class="p-strip is-shallow">
+    <div class="row">
+        <div class="col-4 col-medium-3">
+            <h4>Why not try widening your search?<br> You can do this by:</h4>
```

Comment:
Just noticed also this jumps from a h2 > h4. You could use a <h3> but add the p-heading--4 classname

Reply:
This is going to be fixed in [this PR](https://github.com/canonical-web-and-design/vanilla-framework/pull/4276#pullrequestreview-866278371)